Add `SelectFromQuadraticModel` class (#5)
dwavesystems/dwave-scikit-learn-plugin: README.md

[![PyPI](https://img.shields.io/pypi/v/dwave-scikit-learn-plugin.svg)](https://pypi.python.org/pypi/dwave-scikit-learn-plugin)
[![CircleCI](https://dl.circleci.com/status-badge/img/gh/dwavesystems/dwave-scikit-learn-plugin/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/dwavesystems/dwave-scikit-learn-plugin)

# D-Wave `scikit-learn` Plugin

> This package provides a `scikit-learn` transformer for 

Reviewer: ```suggestion
This package provides a [scikit-learn](https://scikit-learn.org/stable/) transformer for 
```

[feature selection](https://en.wikipedia.org/wiki/Feature_selection) using a quantum-classical hybrid solver. 

For an introduction to hybrid methods for feature selection, see the [Feature Selection example](https://github.com/dwave-examples/feature-selection-notebook) Jupyter Notebook.

> The package's main class, `sklearn.feature_selection.SelectorMixin`, can be used in any existing `sklearn` pipeline.

Reviewer: ```suggestion
The package's main class, `SelectFromQuadraticModel`, can be used in any existing `sklearn` pipeline.
```

## Examples

A minimal example of using the plugin: 

```python
from dwave.plugins.sklearn.transformers import SelectFromQuadraticModel
import numpy as np

# generate uniformly random data, 10,000 observations and 300 features
data = np.random.uniform(-10, 10, size=(10000, 300))

outcome = np.array([int(i) for i in (np.random.uniform(0, 1, size=(10000, 1)) > .5)])

# instantiate the feature selection class
selector = SelectFromQuadraticModel()

# solve the feature-selection problem

> data_transformed = selector.fit_transform(data, outcome)

Reviewer: I no longer see a `fit_transform` method

```

> For large problems, the default runtime may be insufficient. You can use the CQM solvers's 

Reviewer: ```suggestion
For large problems, the default runtime may be insufficient. You can use the CQM solver's 
```
